### ISSUE TITLE
import mill project on MacOS 10.15 with Metals 0.8.0 doesn't work

### ISSUE THREAD

**Issue by tylerprete** [scalameta/metals-vscode#175]
**Describe the bug**
When attempting to import a simple mill project with metals 0.8.0 in vscode, I'm getting "import project failed, no functionality will work."

Appears to be attempting to write to /.metals/download instead of my home directory, which is odd, because in metals/src/main/resources/millw the download path is set like so:
```
MILL_DOWNLOAD_PATH="${HOME}/.mill/download"
```

Perhaps it can't pick up $HOME in the context it's running in?

**To Reproduce**
1. Download my super simple mill project (or create one): https://github.com/tylerprete/little-mill
2. Attempt vscode/metals import from cmdline with ```code .``` in checkout folder.
3. Run "Metals: Import Build" manually if VSCode doesn't suggest it automatically
3. See error

**Expected behavior**
Metals is able to download mill successfully and setup bloop.

**Screenshots**

**Installation:**
- Operating system: macOS Catalina (10.15)
- Editor: Visual Studio Code
- Metals version: v0.8.0

**Additional context**
metals.log contain this:
```
INFO  started: Metals version 0.8.0 in workspace '/Users/tprete/repos/little-mill'
INFO  time: initialize in 0.32s
INFO  running '/var/folders/tp/p9cbzcjj5nb268sl3bnth19w0000gn/T/metals5540375500728896923/millw --mill-version 0.5.2 --predef /var/folders/tp/p9cbzcjj5nb268sl3bnth19w0000gn/T/metals5540375500728896923/predef.sc mill.contrib.Bloop/install'
INFO  Downloading mill 0.5.2 from https://github.com/lihaoyi/mill/releases ...
INFO    % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
INFO                                   Dload  Upload   Total   Spent    Left  Speed
100   605    0   605    0     0   1563      0 --:--:-- --:--:-- --:--:--  1563
100 49.9M  100 49.9M    0     0  22.1M      0  0:00:02  0:00:02 --:--:-- 41.9M
INFO  mkdir: /.mill/download: Read-only file system
INFO  build tool exit: 1
INFO  time: ran 'mill bloopInstall' in 2.48s
ERROR Mill command failed: /var/folders/tp/p9cbzcjj5nb268sl3bnth19w0000gn/T/metals5540375500728896923/millw --mill-version 0.5.2 --predef /var/folders/tp/p9cbzcjj5nb268sl3bnth19w0000gn/T/metals5540375500728896923/predef.sc mill.contrib.Bloop/install
INFO  shutting down Metals
```

**Search terms**
failed mill project import

**Comment by gabro** [comment]
I've transferred this to metals-vscode, since it's a vscode-specific issue

**Comment by jodersky** [comment]
#174 should fix this specific issue.

However, do note that you'll need to use an old version of mill (<= 0.5.3, I think) to get the full experience, as bloop integration is currently broken (for reasons unrelated to this issue). Hopefully once https://github.com/lihaoyi/mill/pull/752 is in, mill should once again be a first-class citizen in the metals ecosystem.

**Comment by gabro** [comment]
Fix released as 1.7.2
